### PR TITLE
Remove Learning Outcomes section #82

### LINKED ISSUE #82: Remove Learning Outcomes section
Contents of LO section will be superseded by the text book. Can be removed.

### PR DESCRIPTION
**Pull request by limmlingg**
Fixes #82

### CODE REVIEW

**Comment by Zhiyuan-Amos** [comment]
I'm not familiar with Address Book Level 1, perhaps @yamgent can help

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
   the main programming language. 
 * It provides a **reasonably well-written** code example that is **significantly bigger** than what students 
   usually write in data structure modules. 
-* It can be used to achieve a number of beginner-level [learning outcomes](#learning-outcomes) without 
-  running into the complications of OOP or GUI programmings.
+* It can be used to achieve a number of beginner-level learning outcomes without running into the complications
```
Comment (yamgent) [inline README.md]: Perhaps we can change it to link to the new textbook's learning outcomes, unless that is not ready yet? @damithc 

Also a very minor issue: We do not need to maintain the wrapping for documentation, it creates unnecessary noise in the diff.

Comment (damithc) [inline README.md]: Text book LOs are not ready for linking yet. We can do that later.

**Comment by damithc** [comment]
As discussed, let's put this on hold for now. I'll whittle down the LO description at a later time.

**Pull request closed without merging**